src/components/organisms/systemLog.tsx

```diff
@@ -8,12 +8,35 @@ import { cn } from "@src/utilities";
 
 import { Frame, IconButton, Typography } from "@components/atoms";
 
-import { Close, TrashIcon } from "@assets/image/icons";
+import { Close, TrashIcon, ArrowUpIcon } from "@assets/image/icons";
 
 export const SystemLog = () => {
 	const { clearLogs, logs, setSystemLogHeight } = useLoggerStore();
 	const { t } = useTranslation("projects", { keyPrefix: "outputLog" });
 
+	const logContainerRef = useRef<HTMLDivElement>(null);
```

Comment:
I checked locally, and it seems that the scroll is not working. I performed multiple actions, but when opening the system log, it does not scroll to the top and the "scroll up" arrow is not displayed.
I’ll try to attach a video later for reference.